Modules/Loadable/Sequences/Widgets/qMRMLSequenceBrowserPlayWidget.cxx

```diff
@@ -307,23 +307,23 @@ void qMRMLSequenceBrowserPlayWidget::setPlayPauseShortcut(QString keySequence)
 {
   Q_D(qMRMLSequenceBrowserPlayWidget);
   QObject::connect(new QShortcut(QKeySequence(keySequence), this), SIGNAL(activated()), SLOT(onVcrPlayPause()));
-  d->pushButton_VcrPlayPause->setToolTip(d->pushButton_VcrPlayPause->toolTip()+" ("+keySequence+")");
+  d->pushButton_VcrPlayPause->setToolTip("Play/Pause (" + keySequence + ")");
```

Comment:
Messages that are displayable to users cannot be concatenated (because order of words is different in each language), so always placeholders need to be used. Something like this:

```suggestion
  d->pushButton_VcrPlayPause->setToolTip(tr("Play/Pause (%1)").arg(keySequence));
```

Please update everywhere else.

Reply:
I've pushed the according updates to have the translation support.